packages/adapters-library/src/scripts/buildMetadataDb.ts

```diff
@@ -155,6 +156,31 @@ async function buildAdapterMetadata(adapter: IProtocolAdapter) {
     return
   }
 
+  const invalidPoolAddress = metadataDetails.find((pool) => {
+    if (pool.address === ZERO_ADDRESS) {
+      return true
+    }
+  })
+
+  if (invalidPoolAddress) {
+    console.error(chalk.yellow(ZERO_ADDRESS))
```

Comment:
```suggestion
    console.error(chalk.yellow(invalidPoolAddress.address))
```
A bit unsafe not to use the address of the pool returned otherwise, just in case we change the condition.